src/ralph/back_office/helpers.py

```diff
@@ -10,3 +15,36 @@ def get_email_context_for_transition(transition_name: str) -> EmailContext:
         'body': 'Please see documents provided in attachments for "{}".'.format(transition_name)  # noqa
     }
     return EmailContext(**default)
+
+
+def status_converter(
+    old_status_id, target_status_id, from_status_cls=BackOfficeAssetStatus,
```

Comment:
I think it would be good to add some assert to check if `from_status_cls` and `to_status_cls` are one of expected classes. It's easy to miss description in dcostring.

Reply:
I was considering a flag (maybe `reverse`?) that would specify from what type of status to what type of status to convert and then basing on that flag, set from_status_cls and to_status_cls inside the method. Default would be from BackOfficeAssetStatus to DataCenterAssetStatus, and `reverse` would be the other way round. What do you think?

Reply:
I can suggest to make this method internal and create two explicit wrappers for both cases that will have the params hardcoded.